src/command-registry.coffee

```diff
@@ -175,11 +175,14 @@ class CommandRegistry
   # * `commandName` {String} indicating the name of the command to dispatch.
   dispatch: (target, commandName, detail) ->
     event = new CustomEvent(commandName, {bubbles: true, detail})
-    eventWithTarget = Object.create event,
+    eventWithTarget = Object.create {},
       target: value: target
       preventDefault: value: ->
       stopPropagation: value: ->
       stopImmediatePropagation: value: ->
+    # In Chrome 43, Object.create doesn't work well with CustomEvent.
+    for k, v of event when k not in eventWithTarget
+      eventWithTarget[k] = v
```

Comment:
This whole block needs to change anyway, since all properties have been moved into the prototypes, which is why added e3174cd. I'll need to shadow the prototype properties from `CustomEvent` (only one IIRC) and `Event`.